src/shared_utilities/drivers.jl

```diff
@@ -289,6 +289,43 @@ function turbulent_fluxes!(
 end
 
 
+function coupler_compute_turbulent_fluxes!(
+    dest,
+    atmos::NamedTuple,
```

Comment:
Will `atmos` be the atmos model state for this function? I assume so because where this is called we pass in the atmos state, but then in this function we access `atmos.thermal_state`, `atmos.u`, `atmos.h`, etc. which I wouldn't be present in the atmos state.

In the coupler, before we compute fluxes we retrieve from the atmosphere `u`, surface height `z_sfc`, first atmos center height `z_int`, and atmos thermo state `thermo_state_int`, then pass these into the functions that compute fluxes. Maybe here, `atmos` could instead be a NamedTuple `atmos_properties` or something like that, which contains these 4 properties.

Reply:
With the changes in https://github.com/CliMA/ClimaCoupler.jl/pull/1220, this function will work as-is